core/common/assets/js/api/core/commands.js

```diff
@@ -1,4 +1,8 @@
+import CommandBase from 'elementor-api/modules/command-base';
```

Comment:
It shouldn't be `command`?

Reply:
**CommandBase** is the base class parent of all commands, later we validate if the command is instance of **CommandBase**.